static/scripts/homework.js

```diff
@@ -445,4 +445,39 @@ $(document).ready(() => {
             });
         });
     });
+
+    $('a[data-method="delete-file-homework-edit"]').on('click', function (e) {
+        e.stopPropagation();
+        e.preventDefault();
+        let $buttonContext = $(this);
+        let $deleteModal = $('.delete-modal');
+        let fileId = $buttonContext.data('file-id');
+
+        $deleteModal.appendTo('body').modal('show');
+        $deleteModal.find('.modal-title').text("Bist du dir sicher, dass du '" + $buttonContext.data('file-name') + "' löschen möchtest?");
+
+        $deleteModal.find('.btn-submit').unbind('click').on('click', function () {
+            $.ajax({
+                url: $buttonContext.attr('href'),
+                type: 'DELETE',
+                data: {
+                    key: $buttonContext.data('file-key')
+                },
+                success: function (_) {
```

Comment:
_ und in 474, da sollte der linter wegen unused vars meckern.

Reply:
done